### PR TITLE
Cloudinary gabi

### PR DESCRIPTION
**Pull request by gwittm**

### CODE REVIEW

**Comment by gwittm** [comment]
It worked yesterday - at least at localhost. Now vercel and localhost give us errors... we try to solve the problem :/

**Comment by doemser** [comment]
Then you need to add the enviromental variables for cloudinary to vercel also.

**Comment by gwittm** [comment]
Done ;-)

File: pages/recipes/[id]/index.js [hunk 1]
```diff
@@ -53,43 +60,40 @@ export default function DetailsPage() {
...
+              xmlns="http://www.w3.org/2000/svg"
+              height="1em"
+              viewBox="0 0 448 512"
+            >
+              <path
+                d="M135.2 17.7L128 32H32C14.3 32 0 46.3 0 64S14.3 96 32 96H416c17.7 0 32-14.3 32-32s-14.3-32-32-32H320l-7.2-14.3C307.4 6.8 296.3 0 284.2 0H163.8c-12.1 0-23.2 6.8-28.6 17.7zM416 128H32L53.2 467c1.6 25.3 22.6 45 47.9 45H346.9c25.3 0 46.3-19.7 47.9-45L416 128z"
+                fill="#e9f0ef"
+              />
+            </svg>
+          </span>
```
Comment (doemser) [inline pages/recipes/[id]/index.js:96]: you should not need the span here

File: components/ImageUpload/StyledImageUpload.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  margin: 10px;
+  padding: 10px;
+`;
+
+export const StyledImageButtonReset = styled.button`
+  border: solid 1px black;
+  border-radius: 5px;
+  margin: 10px;
+  padding: 10px;
+`;
```
Comment (doemser) [inline components/ImageUpload/StyledImageUpload.js:43]: This styles are the same, so you could use one button for botch use-cases

**Review by doemser — CHANGES_REQUESTED** [review]
Looks good!

Just try to reuse the button style.

**Comment by gwittm** [comment]
thank you for your comments, we changed both :)

**Review by doemser — APPROVED** [review]

**Pull request merged**